get_trust_anchor.py

```diff
@@ -80,15 +73,6 @@
 -----END CERTIFICATE-----
 '''
 
-URL_ROOT_ANCHORS = "https://data.iana.org/root-anchors/root-anchors.xml"
```

Comment:
All the constants should be in main() so they are visible to a reader. They are just as editable there as they are at the top of the program. However, it is better to put the ICANN cert at the top of the program to show that it is built in, not retrieved.